### PR TITLE
Improved performance of counters by caching results

### PR DESCRIPTION
**Pull request by Markus92**
Hi there,
We added some caching to the portal and dataset statistics. This will cache the returned values from the CKAN back-end for 24 hours.
Works on Firefox, Chrome and Edge. Unfortunately it does not work for Safari, it will ignore the caching parameter and act like the previous version.

The commit also adds sharp for images to the package, to make `npm run build` a bit happier ;)

### CODE REVIEW

File: src/services/ckan/datasetCount.ts [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-import axios from 'axios';
+// import axios from 'axios';
 import { constructCkanActionUrl } from './utils';
 
 export const makeDatasetCount = (DMS: string) => {
   return async (): Promise<number> => {
     const url = constructCkanActionUrl(DMS, 'dataset_list');
-    const response = await axios.get(url);
-    return response.data.result.count;
+    const raw_response = await fetch(url, { cache: 'force-cache', next: { revalidate: 24 * 3600 } });
```
Comment (brunopacheco1) [inline src/services/ckan/datasetCount.ts:12]: important: move all literals `24 * 3600` to environment variable configuration, like `CKAN_BACKEND_CACHE_TTL` or something more readable, and default it to 24h.

I don't know Next.js enough, but you should pay attention that this env variable may be accessible on client and server rendereded components.

**Review by brunopacheco1 — CHANGES_REQUESTED** [review]
Can you also please fix you commit messages to follow https://www.conventionalcommits.org/en/v1.0.0/?

File: src/services/ckan/datasetCount.ts [hunk 2]
```diff
@@ -1,14 +1,16 @@
 // SPDX-FileCopyrightText: 2024 PNED G.I.E.
+// SDPX-Contributor: Stichting Health-RI
 //
 // SPDX-License-Identifier: Apache-2.0
 
-import axios from 'axios';
+// import axios from 'axios';
```
[the remark this answers is not in the document]
Comment (hcvdwerf) [reply src/services/ckan/datasetCount.ts]: removed

File: src/services/ckan/datasetCount.ts (hunk 1 above)
Comment (hcvdwerf) [reply src/services/ckan/datasetCount.ts]: I have added a const in top of the file. We think an environment variable is too verbose? Agree?

Comment (brunopacheco1) [reply src/services/ckan/datasetCount.ts]: I was thinking on somewhere central place and reuse a single variable on both pleaces. But indeed perhaps it's not needed now.

**Comment by brunopacheco1** [comment]
@Markus92 @hcvdwerf can you please squash and update the commit message?

https://www.conventionalcommits.org/en/v1.0.0/

**Comment by hcvdwerf** [comment]
> @Markus92 @hcvdwerf can you please squash and update the commit message?
> 
> https://www.conventionalcommits.org/en/v1.0.0/

done

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**